ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -47,6 +47,8 @@ open class SegmentedControl: UIControl, TokenizedControlInternal {
             invalidateIntrinsicContentSize()
         }
     }
+    /// Action called when a segment is selected.
+    @objc public var onSelectAction: ((SegmentedControl, SegmentItem, Int) -> Void)?
```

Comment:
better documentation what int means? but i don't quite understand why we need segmentedcoontrol as a param?

Reply:
The SegmentedControl is in there so it could be used in the same ways as sender used to. We could probably remove it, but our demo control uses the SegmentedControls to access the labels below them.